packages/garbo/src/session.ts

```diff
@@ -309,7 +309,7 @@ export function endSession(printLog = true): void {
     message("So far today", totalTurns, totalMeat, totalItems);
 
     printMarginalSession();
-    if (globalOptions.quick) {
+    if (isAnyQuick(globalOptions)) {
```

Comment:
I'm not sure this is still true, because of the historical price thing. we should just remove this messaging